.gitignore

```diff
@@ -12,7 +12,7 @@ target/
 .settings
 .springBeans
 .sts4-cache
-
+!**/src/main/resources/log4j2.xml
```

Comment:
we need to also commit this as well, otherwise how will the person who clones your project will able to change the configs